cmd/container-use/stdio_test.go

```diff
@@ -0,0 +1,359 @@
+package main_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/mark3labs/mcp-go/client"
+	"github.com/mark3labs/mcp-go/mcp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// MCPServerProcess represents a running container-use MCP server
+type MCPServerProcess struct {
+	cmd        *exec.Cmd
+	client     *client.Client
+	repoDir    string
+	configDir  string
+	serverInfo *mcp.InitializeResult
+	t          *testing.T
+}
+
```

Comment:
🐛 **Bug (Medium):** Should include build output in error message for better debugging (requires implementing the stderr/stdout capture suggestion above)

```suggestion
if err != nil {
			t.Fatalf("Failed to build container-use binary: %v\nStderr: %s\nStdout: %s", err, stderr.String(), stdout.String())
		}
```